client/components/command-palette/generate-command-palette-interface-link.ts

```diff
@@ -0,0 +1,17 @@
+import { SiteExcerptData } from 'calypso/data/sites/site-excerpt-types';
+import { siteDefaultInterface } from 'calypso/sites-dashboard/utils';
+
+export const generateSiteInterfaceLink = (
+	site: SiteExcerptData,
+	calypsoBase: string,
+	wpAdminBase: string
```

Comment:
I'd be tempted to combine `calypsoBase` and `wpAdminBase` into one argument. At the moment, it's somewhat confusing as to which comes first.

Reply:
Let's combine `calypsoBase` and `wpAdminBase` in one single object 👌

Reply:
renamed into one single argument https://github.com/Automattic/wp-calypso/pull/84864/commits/9530106c62c40751dea01221753cca989ca19325